qcodes/tests/dataset/test_dataset_loading.py

```diff
@@ -123,11 +138,14 @@ def test_completed_timestamp_for_not_completed_dataset():
     _ = new_experiment(name="for_loading", sample_name="no_sample")
     ds = new_data_set("my_first_ds")
 
-    assert False is ds.completed
+    assert ds.pristine is True
+    assert ds.started is False
+    assert ds.running is False
+    assert ds.completed is False
 
-    assert None is ds.completed_timestamp_raw
+    assert ds.completed_timestamp_raw is None
 
-    assert None is ds.completed_timestamp()
+    assert ds.completed_timestamp() is None
```

Comment:
you'd be surprised but the reason to write `assert None is ...` is that for some reason the expected order is `assert expected is actual`, not `assert actual is expected`, as per how pytest creates this insight into the compared values.